cgmanifest.json

```diff
@@ -3188,8 +3188,8 @@
         "type": "other",
         "other": {
           "name": "fish",
-          "version": "3.1.2",
-          "downloadUrl": "https://github.com/fish-shell/fish-shell/releases/download/3.1.2/fish-3.1.2.tar.gz"
+          "version": "3.5.0",
```

Comment:
Are there any packages that depend on fish that need to be bumped as well...given the large version jump from 3.1 to 3.5.  (Theoretically this is ok...but just want to double check)

Reply:
Updated two packages (one is in specs-extended which I don't have a system setup to build quickly, so untested).
gh builds fine with the new fish